test/acceptance/features/steps/openshift.py

```diff
@@ -334,3 +335,14 @@ def search_resource_lst_in_namespace(self, resource_plural, name_pattern, namesp
             return lst
         print('Resource list is empty under namespace - {}'.format(namespace))
         return None
+
+    def get_secret_info_custom_env_var(self, secret_name, namespace):
+        oc_cmd = f'oc get secrets {secret_name}  -n {namespace} -o "jsonpath={{.data.CustomReady}}"'
```

Comment:
Hi @pedjak I am unsure as to how to pass a parameter to the jsonPath . like jsonpath={{.data.$(var name)}} . Please suggest how to pass the var name there. I see the data to be fetched being directly used, like https://github.com/redhat-developer/service-binding-operator/blob/master/test/acceptance/features/steps/openshift.py#L197 . It would be good to have a param key . Not sure on how to achieve it

Reply:
@Avni-Sharma Please refer to this comment: https://github.com/redhat-developer/service-binding-operator/pull/597#discussion_r481769679

Reply:
Checked. What I am unsure is that the function there takes a jsonPath variable which in my case will be jsonpath={.data.CustomReady} but how can I pass the CustomReady as a varibale like I tried {.data.${custom_name}} but it didn't work

Reply:
@Avni-Sharma I was thinking about the following:

```python
def get_secret_key_value(self, secret_name, namespace, key):
  oc_cmd = f'oc get secrets {secret_name}  -n {namespace} -o "jsonpath={{.data.{key}}}"'
```

Reply:
Thanks @pedjak I will give this a try

Reply:
Done https://github.com/redhat-developer/service-binding-operator/pull/597/commits/b904a2186b93b06ef06213f2a23139572628abdc